.gitlab/source_test/ebpf.yml

```diff
@@ -64,13 +62,13 @@ tests_ebpf_x64:
     # Compile runtime security stress tests to be executed in kitchen tests
     - inv -e security-agent.build-stress-tests --output=$DD_AGENT_TESTING_DIR/site-cookbooks/dd-security-agent-check/files/stresssuite
     # Compile main version for comparison, uncomment following lines when merged
-    - git checkout -f main || git checkout -f master
-    - git pull
-    - inv -e deps
-    - inv -e system-probe.build --bundle-ebpf --incremental-build
-    - inv -e security-agent.build-stress-tests --output=$DD_AGENT_TESTING_DIR/site-cookbooks/dd-security-agent-check/files/stresssuite-master
-    - git reset --hard
-    - git checkout -
+    # - git checkout -f main || git checkout -f master
```

Comment:
This was commented out in Sylvain's original PR but you're right that it shouldn't be

Reply:
Hmm now I know, since we are removing bcc we cannot build the main branch (because it requires bcc) so we actually have to do it in two steps:
- Comment and merge
- Uncomment in another PR where main will compile without bcc
Job failing related to that: https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/80828359